src/blocks/scratchpad.rs

```diff
@@ -0,0 +1,92 @@
+//! Scratchpad indicator
+//!
+//! # Configuration
+//!
+//! Key | Values | Default
+//! ----|--------|--------
+//! `format`          | A string to customise the output of this block      | `" $icon $count "`
+//! `hide_when_empty` | Hides the block when scratchpad contains no windows | `true`
```

Comment:
Technically, `hide_when_empty` is not needed, because the same effect can be achieved using

```toml
format = " $count.range(1..) |"
```

Reply:
Good to know, the code was updated